modules/core/include/opencv2/core/persistence.hpp

```diff
@@ -1345,6 +1346,19 @@ inline String::String(const FileNode& fn): cstr_(0), len_(0) { read(fn, *this, *
 
 //! @endcond
 
+/** @brief Loads an object from a filestorage.
+
+This performs the same functionality as @cvLoad, but does it directly from a FileStorage object.
+@param filename File storage object
+@param memstorage Memory storage for dynamic structures, such as CvSeq or CvGraph . It is not used
+for matrices or images.
+@param name Optional object name. If it is NULL, the first top-level object in the storage will be
+loaded.
+@param real_name Optional output parameter that will contain the name of the loaded object
+(useful if name=NULL )
+ */
+CV_EXPORTS void* cvLoadFileStorage(FileStorage& filestorage, CvMemStorage* memstorage, const char* name, const char** real_name);
```

Comment:
Adding of new public entries for legacy C-like API must be avoided.
CvMemStorage and other "C API" things would be removed in OpenCV 4.0, so nobody would write proper tests for this.